### PR TITLE
feat(Heading): default level is now 2

### LINKED ISSUE #2358: Change default Heading level to 2 as this is most used

### PR DESCRIPTION
**Pull request by Barsnes**
resolves #2358

### CODE REVIEW

**Review by mimarz — CHANGES_REQUESTED** [review]
Did you forget to update the actual default value (L34)? 😅

File: packages/react/src/components/Typography/Heading/Heading.tsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ export type HeadingProps = {
  */
 export const Heading = forwardRef<HTMLHeadingElement, HeadingProps>(
   (
-    { size = 'xl', level = 1, spacing = false, className, asChild, ...rest },
+    { size = 'xl', level = 2, spacing = false, className, asChild, ...rest },
     ref,
   ) => {
     const Component = asChild ? Slot : (`h${level ?? 1}` as ElementType);
```
Comment (mimarz) [inline packages/react/src/components/Typography/Heading/Heading.tsx:37]: Why do we even have this fallback check here?

Comment (Barsnes) [reply packages/react/src/components/Typography/Heading/Heading.tsx:37]: Not sure, I can remove it

**Comment by eirikbacker** [comment]
Nice! ☺️ 
- should we then update everywhere `Heading level={2}` is used?
- should we double check so everything where `Heading` does not have `level` specified (as this is now changes level)?

**Comment by Barsnes** [comment]
> Nice! ☺️
> 
> * should we then update everywhere `Heading level={2}` is used?
> * should we double check so everything where `Heading` does not have `level` specified (as this is now changes level)?

I think it's fine if we keep `level={2}`, it makes a bit more readable for everyone :)

I did a quick check through places we have used `Heading`, and it seems like we set the level most of the time. Testsites we don't need to update.

**Review by eirikbacker — APPROVED** [review]

**Review by mimarz — APPROVED** [review]

**Pull request merged**